MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+
+If you are using the "indexName" option from SCK you can use the index option in Splunk OpenTelemetry Collector for Kubernetes to specify which index you want to index data into. Translating custom configs from SCK to Splunk OpenTelemetry Collector for Kubernetes for logs
+
+## Translating custom configs from SCK to Splunk OpenTelemetry Collector for Kubernetes for logs
+You can find all the configuration options used to upgrade in the values.yaml files linked above for both SCK and Splunk OpenTelemetry Collector for Kubernetes.
+ 
+#### Using Root user/permissions for accessing log files
+If you are using root user/permissions for accessing log files in SCK, you can do so by setting "runAsUser: 0" in the securityContext in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Container runtimes
+If you configured SCK to explicitly set any of the following, you can do the same with Splunk OpenTelemetry Collector for Kubernetes:
+
+* Set the **containerRuntime** option to a value from (CRI-O, containerd and docker) depending on your runtime.
+* Set the **path** option to the location of your logs on your nodes.
+* If you are using the **exclude_path** option from SCK, you can use the **excludePaths** option in Splunk OpenTelemetry Collector for Kubernetes to exclude any logs you want.
+* If you are using the **"logs"** option to define **"multiline**" configs in SCK, you can use the **multilineConfigs** option in Splunk OpenTelemetry Collector for Kubernetes to concatenate multiline logs. 
+* If you are using the **checkpointFile** option in SCK to define a custom location for checkpointing your logs, you can also do so with the **checkpointPath** option in Splunk OpenTelemetry Collector for Kubernetes.
+* If you are using the **"logs"** option to ingest any other logs from your nodes, you can also do so with the **extraFileLogs** option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Cluster Name
+If you are using the clusterName option to set a cluster name metadata field in your logs in SCK, you can also use the clusterName option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Pod Security Context
+If you are using the podSecurityContext option to set a pod security policy, you can use the podSecurityContext option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Custom Metadata
+If you are using the customMetadata option to set a custom metadata field in your logs in SCK, you can use the customMetadata option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Custom Annotations
+If you are using the customMetadataAnnotations option to set custom annotation fields in your logs in SCK, you can use the "annotations" and "podAnnotations" option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Pod Scheduling configs
```

Comment:
"Pod scheduling configurations"

" ... nodeSelector, affinity, and tolerations ..."